doc/install.rst

```diff
@@ -61,17 +61,23 @@ If you want to upgrade from an existing previous version, replace
 ``install`` by ``ìnstall --upgrade`` in the above commands.
 
 
-Install optional packages
--------------------------
+Install optional packages and external dependencies
+---------------------------------------------------
 
-* This package includes multiple comfort methods simplyfing its use for
+* This package includes multiple comfort methods simplifying its use for
```

Comment:
```suggestion
* pyPESTO includes multiple convenience methods to simplify
```

Usually "convenience functions", though perhaps "convenience methods" or "helper methods" is more appropriate for Python.